### PR TITLE
Create ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58016](https://cumulocity.atlassian.net/browse/MTM-58016)
Original PR: [6435](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6435)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Body text in the confirmation modal for logging out all users is not translated (#6281) [GRAFT][release/y2024] (#6435)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Confirmation modal for logging out all users now has translated body text
```

File: content/change-logs/platform-services/ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58016
+version: 1018.503.90
+---
+Body text in the confirmation modal for logging out all users is not translated (#6281) [GRAFT][release/y2024] (#6435)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT platform, there is a confirmation modal that appears when an administrator attempts to log out all users. Previously, the body text in this modal was not properly translated and always appeared in English, regardless of the user's language settings. With this change, the body text in the confirmation modal is now correctly translated based on the user's selected language. This improves the user experience for non-English speaking users by providing a fully localized interface when logging out all users.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by Airdroplet — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58016
+version: 1018.503.90
+---
+In the Cumulocity IoT platform, there is a confirmation modal that appears when an administrator attempts to log out all users. Previously, the body text in this modal was not properly translated and always appeared in English, regardless of the user's language settings. With this change, the body text in the confirmation modal is now correctly translated based on the user's selected language.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md:17]:
```suggestion
In the Administration application, a confirmation modal appears when an administrator attempts to log out all users. Previously, the body text in this modal was not properly translated and always appeared in English, regardless of the user's language settings. With this change, the body text in the confirmation modal is now correctly translated based on the user's selected language.
```

File: content/change-logs/platform-services/ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Confirmation modal for logging out all users now translated correctly
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md:3]:
```suggestion
title: Confirmation message on logging out all users now translated correctly
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**